### DISCUSSION TITLE
stablediffusion.cpp based on GGML

### DISCUSSION THREAD

**Discussion by zhouwg** [ggml-org/ggml#786] [Ideas]
https://github.com/Stability-AI/stablediffusion

There is already the awesome Whisper.cpp and the amazing LLAMA.cpp, is there a roadmap for a StableDiffusion.cpp based on GGML?

**Comment by Green-Sky** [comment]
https://github.com/leejet/stable-diffusion.cpp

**Reply by Green-Sky** [reply]
it's properly listed in the readme: https://github.com/ggerganov/ggml?tab=readme-ov-file#updates

**Reply by zhouwg** [reply]
thanks so much.
